rfc/3/index.md

```diff
@@ -0,0 +1,158 @@
+# RFC-3: more dimensions for thee
+
+Remove restrictions on the number of dimensions stored in OME-NGFF arrays.
+
+## Status
+
+Draft (D3).
+
+| Name      | GitHub Handle | Institution | Date       | Status                                 |
+|-----------|---------------|-------------|------------|----------------------------------------|
+| Davis Bennett | [d-v-b](https://github.com/d-v-b) | N/A | 2024-05-01 | [Implementing][implementation] |
+| Juan Nunez-Iglesias | [jni](https://github.com/jni) | Monash University | 2024-05-21 | Author |
+| Talley Lambert | [tlambert03](https://github.com/tlambert03) | Harvard Medical School | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issuecomment-2122795327) | -->
+| Norman Rzepka | [normanrz](https://github.com/normanrz) | Scalable Minds | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Davis Bennett | [d-v-b](https://github.com/d-v-b) |  | 2024-05-21 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Doug Shepherd | [dpshepherd](https://github.com/dpshepherd) | Arizona State University | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| John Bogovic | [bogovicj](https://github.com/bogovicj) | HHMI Janelia Research Campus | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Eric Perlman | [perlman](https://github.com/perlman) |  | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+| Lachlan Deakin | [LDeakin](https://github.com/LDeakin) | Australian National University | 2024-05-22 | [Endorse](https://github.com/ome/ngff/pull/239#issue-2308436425) | -->
+
+## Overview
+
+OME-NGFF version 0.4 places severe restrictions on the number, names, and types
+of axes that are allowed in the axes metadata. This has had the effect of
+limiting the datasets in proprietary formats that *can* be meaningfully
+converted to NGFF. It has also prevented some novel datasets from being written
+in NGFF format. This RFC only removes restrictions from the specification. An
+important consequence is that all valid NGFF datasets would remain valid after
+this change.
+
+## Background
```

Comment:
I think it's important to mention here that as a historical fact OME-NGFF is an OME project, and the OME data model is 5D. This I think goes a long way to explaining the intertia for the 5D limit and the other various axis restrictions.

Reply:
I don't think I need to really get into it. I'm happy to change "unfortunately, 0.4 imposes restrictions..." to "unfortunately, **for historical reasons**, 0.4 imposes restrictions...".

Reply:
I don't think the word "unfortunate" fits here -- it's just an explanatory fact that this format was jump-started by OME, and they use a 5D model. This is literally the background for the 5D limit, so it might be helpful to put that here for people who don't know the history.